contracts/signatures/NonceVerifiable.sol

```diff
@@ -19,9 +19,9 @@ abstract contract NonceVerifiable is OwnableUpgradeable {
     /// @custom:schema (contract address -> token id -> signer address -> nonce)
     mapping(address => mapping(uint256 => mapping(address => uint256))) private assetNonce;
 
-    event ContractNonceUpdated(uint256 _from, uint256 _to, address _sender);
-    event SignerNonceUpdated(uint256 _from, uint256 _to, address _signer, address _sender);
-    event AssetNonceUpdated(uint256 _from, uint256 _to, address _contractAddress, uint256 _tokenId, address _signer, address _sender);
+    event ContractNonceUpdated(address indexed _sender, uint256 _newNonce);
```

Comment:
I would put the sender at the end of the event without indexing it as we have in these events https://github.com/decentraland/rentals-contract/blob/main/contracts/Rentals.sol#L103